### PR TITLE
Adds XNAT data request form for first-line information collection

### LINKED ISSUE #174: XNAT data request form to be added to the website
XNAT data request form to be added to the website so we can collect and record requests efficiently between us, allowing oversight and awareness. The form should be emailed to CSC email until we set up XNATadmins usergroup.

### PR DESCRIPTION
**Pull request by dangerdika**
Updates XNAT description and adds XNAT flow diagram of connectivity

### CODE REVIEW

File: _platforms/xnat.md [hunk 1]
```diff
@@ -5,14 +5,30 @@ title: XNAT
 platform-image: /assets/img/platforms/xnat_logo.png
 ---
 
-[XNAT](https://xnat.org/) is an open source imaging informatics platform capable of storing and managing medical images and associated data. Developed by the Neuroinformatics Research Group at Washington University, XNAT can be used to support a wide range of imaging-based projects.
+[XNAT](https://xnat.org/) is an open-source imaging informatics platform for storing and managing medical 
+images and associated data for research purposes. At GSTT we used XNAT to retrieve, anonymise and manage medical imaging
```
Comment (heyhaleema) [inline _platforms/xnat.md]:
``` suggestion
images and associated data for research purposes. At GSTT, we use XNAT to retrieve, anonymise, and manage medical imaging 
```

File: _platforms/xnat.md [hunk 2]
```diff
@@ -5,14 +5,30 @@ title: XNAT
...
+images and associated data for research purposes. At GSTT we used XNAT to retrieve, anonymise and manage medical imaging 
+and non-imaging data for purposes of AI development, AI evaluation, and for data provision for various research projects 
+which are run at GSTT or in collaboration with GSTT. 
 
-Data is ingested from the Picture Archiving and Communications Systems (PACS), the industry-standard storage system for different medical imaging types, into XNAT where it is anonymised (or more accurately, de-identified) and sorted into relevant projects, which ensures the data:
+XNAT at GSTT is currently connected to 3 PACS systems (radiology, foetal cardiology and radiotherapy PACS). Data anonymisation
+processes include DICOM meta-data removal, face blurring and removal of burnt-in data. Data used for AI evaluation are pushed 
+to relevant DICOM nodes using a Python script; data used for AI development are fetched by [MLOPs](https://gstt-csc.github.io/platforms/mlops.html) via
+XNAT's REST-API, and data are sent to external collaborators via pathways agreed in ethics approval of each individual project.
```
Comment (heyhaleema) [inline _platforms/xnat.md]:
``` suggestion
XNAT's [REST API](https://wiki.xnat.org/display/XAPI), and data are sent to external collaborators via pathways agreed in ethics approval of each individual project.
```

File: _platforms/xnat.md [hunk 3]
```diff
@@ -5,14 +5,30 @@ title: XNAT
...
+Using XNAT for data anonymisation and management allows fast and automated retrieval of research data, allows for good quality 
+control and application of highest standards of anonymisation to protect patient privacy. Data visibility is granted explicitly, so 
+only collaborators on a certain project have data visibility. Lastly, access to XNAT is strictly controlled and regularly reviewed. 
 
-For more information, please refer to the [XNAT GitHub repository](https://github.com/GSTT-CSC/XNAT).
+XNAT is only available to GSTT employees with access to GSTT network and before access can be granted an online teaching and 
+training session is held. If you need access to XNAT, please contact the <a href="mailto:CSCTeam@gstt.nhs.uk">CSC team</a>.
+
+A wealth of information, including our SOPs, anonymisation scripts, data retrieval and fetching scripts and similar, please visit our [XNAT GitHub repository](https://github.com/GSTT-CSC/XNAT).
```
Comment (heyhaleema) [inline _platforms/xnat.md]:
```suggestion
For a wealth of information, including our SOPs, anonymisation scripts, data retrieval and fetching scripts and similar, please visit our [XNAT GitHub repository](https://github.com/GSTT-CSC/XNAT).
```

File: _platforms/xnat.md [hunk 4]
```diff
@@ -5,14 +5,30 @@ title: XNAT
...
+processes include DICOM meta-data removal, face blurring and removal of burnt-in data. Data used for AI evaluation are pushed 
+to relevant DICOM nodes using a Python script; data used for AI development are fetched by [MLOPs](https://gstt-csc.github.io/platforms/mlops.html) via
+XNAT's REST-API, and data are sent to external collaborators via pathways agreed in ethics approval of each individual project.
 
-- Is only visible to those who need it
-- Can be deleted upon project completion
-- Is compliant with the [DICOM Standards Supplement 142](https://www.dicomstandard.org/News-dir/ftsup/docs/sups/sup142.pdf)
+<figure>
+<img src="https://github.com/GSTT-CSC/XNAT/blob/main/assets/anon-flowchart.jpg?raw=true" width="100%">
+<figcaption>Schematic representation of XNAT setup at GSTT.</figcaption>
```
Comment (heyhaleema) [inline _platforms/xnat.md:20]: Not super important, but is there a way to make front size of the figure caption smaller (or italicised)? At first glance, it's easy to not notice the distinction between that and the body text.

Screenshot 2023-04-11 at 10 11 52

**Review by heyhaleema — CHANGES_REQUESTED** [review]
@dangerdika Looks great 🔥 I've added a few comments/suggested changes, but they're not required per se e.g., some of them are subjective. 

Once you've resolved the conversations by either taking on the suggested changes or passing on them, I can then go ahead with re-reviewing, approving, and merging.

**Comment by dangerdika** [comment]
@heyhaleema Ive accepted your changes (I agree with them), added them to the commit, and added figcaption specific CSS to render all figcaptions in a smaller and dark grey text to make the distinction clear.

**Review by heyhaleema — APPROVED** [review]

**Pull request merged**